### PR TITLE
CO: debit notes (II)

### PR DESCRIPTION
**Pull request by cavalle**
* Extends #320 
* Part of APP-50

### CODE REVIEW

File: regimes/co/extensions.go [hunk 1]
```diff
@@ -8,8 +8,9 @@ import (
 
 // Extension keys used in Colombia.
 const (
-	ExtKeyDIANMunicipality cbc.Key = "co-dian-municipality"
-	ExtKeyDIANCorrection   cbc.Key = "co-dian-correction"
+	ExtKeyDIANMunicipality     cbc.Key = "co-dian-municipality"
+	ExtKeyDIANCorrectionCredit cbc.Key = "co-dian-correction"
```
Comment (cavalle) [inline regimes/co/extensions.go:12]: We need to separated correction extensions now, one for credit notes and another one for debit notes. I was shy to rename `co-dian-correction` to `co-dian-correction-credit`, but it's probably ok, nothin will break if we do that. @samlown WDYT?

File: regimes/co/extensions.go [hunk 2]
```diff
@@ -91,4 +92,41 @@ var extensionKeys = []*cbc.KeyDefinition{
...
+				Name: i18n.String{
+					i18n.EN: "Interest",
+					i18n.ES: "Intereses",
+				},
+			},
+			{
+				Code: "2",
+				Name: i18n.String{
+					i18n.EN: "Pending charges",
+					i18n.ES: "Gastos por cobrar",
```
Comment (cavalle) [inline regimes/co/extensions.go:113]: The Spanish names are those in the DIAN docs. I'm not 100% sure about the English translation in this particular case.

File: regimes/co/extensions.go (hunk 1 above)
Comment (samlown) [reply regimes/co/extensions.go:12]: oh, that's very annoying... Do you know what the underlying code is called? This key was written some time ago, I'd be more tempted now to use a directly translation of the XML field name.

Comment (cavalle) [reply regimes/co/extensions.go:12]: These are the docs regarding those fields:

image

In both cases, the same UBL field is used, but the list of allowed values and their meaning are different depending on whether the type is a debit or a credit note. That's why I split the extensions in two.

Comment (samlown) [reply regimes/co/extensions.go:12]: I think I'd be tempted to go with simply:
* `co-dian-credit-code`
* `co-dian-debit-code`
Reasoning: "correction" is implied in "credit" and "debit", and "code" makes it clear that this is something defined in a list of codes somewhere.

Comment (cavalle) [reply regimes/co/extensions.go:12]: Updating!

Comment (cavalle) [reply regimes/co/extensions.go]: Address in [88db17e](https://github.com/invopop/gobl/pull/323/commits/88db17e75e12fb4bd503cb91cf4cefa0763fafb8)

File: regimes/co/extensions.go (hunk 2 above)
Comment (samlown) [reply regimes/co/extensions.go:113]: Translation seems good to me.

**Review by samlown — APPROVED** [review]
Looks great. I like the names to, "correction" felt too generic

**Pull request merged**